docs/release-notes.rst

```diff
@@ -6,6 +6,190 @@
  Release Notes
 ###############
 
+**************
+ Version 0.35
+**************
+
+Version 0.35.0
+==============
+
+**Release Date:** August 02, 2024
+
+**Breaking Changes**
+
+-  Master Configuration: Replace ``resource_manager.name`` with ``resource_manager.cluster_name``
+   for better clarity and to support multiple resource managers.
+
+   -  Resource managers operate relative to the specific cluster providing resources for Determined
+      tasks, so changing the cluster will affect the resource manager's responses.
+   -  The ``cluster_name`` must be unique for all resource managers when deploying multiple resource
+      managers in Determined.
+   -  During upgrade, replace ``name`` with ``cluster_name`` in the ``resource_manager`` section of
+      your master configuration YAML.
+
+-  Master Configuration: Replace ``resource_manager.namespace`` with
+   ``resource_manager.default_namespace``.
+
+   -  The namespace field in the Kubernetes Resource Manager configuration is no longer supported
+      and is replaced by ``default_namespace``.
+   -  This field serves as the default namespace for deploying namespaced resources when the
+      workspace associated with a workload is not bound to a specific namespace.
+   -  If unset, the workloads will be sent to the release namespace, or the default Kubernetes
+      namespace, "default", during non-helm determined deployments.
```

Comment:
If unset, the workloads will be sent to the release namespace during determined helm installs or upgrades and will be sent to the default Kubernetes namespace, "default", during non-helm determined deployments.